pkg/codegen/python/gen.go

```diff
@@ -805,7 +807,7 @@ func (mod *modContext) importObjectType(t *schema.ObjectType, input bool) string
 		return "from . import outputs"
 	}
 
-	importPath := mod.getRelImportFromRoot()
+	importPath := mod.getRelImportFromRoot("")
```

Comment:
I suspect we need to fix this and the enum case too. I wanted to see how the resource case works out first.

Reply:
Turns out it works ok for types and enums since they get imported like

```
from ... import mymod
from ... import mymod as _
```